### ISSUE TITLE
ex. 3.2.1 overfitting - parameters only overfit for current noise

### ISSUE THREAD

**Issue by viktorattilakelemen** [ulfaslak/annadl_f19#5]
I'm having an issue where after finally finding parameters for the NN with which it overfits the random data (noise), if I regenerate the data the NN no longer trains to overfit (gets stuck around 50% accuracy). Then I have to regenerate the data many times to end up with noise that the NN can overfit again. Anyone with the same issue? 

(Using 1000 neuron hidden layer, learning rate of 0.05, momentum of 0.9, relu activation, 50 batch size, 10,000 epochs)

**Comment by ulfaslak** [comment]
Ahh sorry I forgot to answer in time! Try a smaller momentum (0 to 1e-1) and a higher learning rate (~1). If that doesn't work something must be going on in your architecture. I've allowed late handin for you, so you can correct it and resubmit if you want.
